code/modules/projectiles/guns/ballistic.dm

```diff
@@ -89,6 +89,7 @@
 	var/flip_cooldown = 0
 	var/suppressor_x_offset ///pixel offset for the suppressor overlay on the x axis.
 	var/suppressor_y_offset ///pixel offset for the suppressor overlay on the y axis.
+	var/hidden_chambered = FALSE // Check if you are able to see if a weapon has a bullet loaded in or not.
```

Comment:
```suggestion
	/// Check if you are able to see if a weapon has a bullet loaded in or not.
	var/hidden_chambered = FALSE 
```
Other vars may not be auto-doc'd, but be sure to any new vars are auto-doc'd regardless.